setup.py

```diff
@@ -1,31 +1,46 @@
+import os
+from pathlib import Path
 from setuptools import setup, find_packages
+from copy import copy
+from shutil import copy as copy_file
 
-d = {}
-exec(open("roiextractors/version.py").read(), None, d)
-version = d["version"]
-pkg_name = "roiextractors"
-with open("README.md") as rd:
-    long_description = rd.read()
+
+path = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I have always wanted to do all of this with `pathlib` instead of `os`

Reply:
Shouldn't be hard, lemme mess with that rq

Reply:
I'll do it in a follow-up, though. Need to get this through ASAP